.github/workflows/release.yaml

```diff
@@ -12,7 +12,21 @@ jobs:
     uses: ./.github/workflows/test.yaml
     secrets: inherit
 
+  publish-release:
+    needs: [tests]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create GitHub Release
+        uses: elgohr/Github-Release-Action@v5
```

Comment:
Prefer softprops/action-gh-release because it drafts up the contents from `.github/release.yml` and the PRs